### ISSUE TITLE
Could you help me?

### ISSUE THREAD

**Issue by KevinKai-0717** [backseason/SCNet#7]
First of all, thank you very much for your open source work!But,I noticed in (scnet.py, SCBottleneck ) :

residual = x
self.conv3 = nn.Conv2d(group_width * 2, planes * 4, kernel_size=1, bias=False)
out = self.conv3(torch.cat([out_a, out_b], dim=1))
out = self.bn3(out)
out += residual
it seems makes out expand 4 scales.However,it(out) directly add to residual（make the tensor is mismatch in dimension 1）.Could you tell me how to solve it?

**Comment by backseason** [comment]
If the channel dimension of `residual` and `out` are not the same, the `residual` part will be forwarded through a `downsample` layer to map its channels the same as `out`. 
https://github.com/MCG-NKU/SCNet/blob/c0b5bd6aa919c00afb5815b2810e645e6a4a5976/scnet.py#L122

The `downsample` layer is defined here:
https://github.com/MCG-NKU/SCNet/blob/c0b5bd6aa919c00afb5815b2810e645e6a4a5976/scnet.py#L212

**Comment by KevinKai-0717** [comment]
Thanks a lot! I will try.
